client/src/components/App.jsx

```diff
@@ -13,10 +13,11 @@ const DarkBG = styled.div`
   background-color: darkslategrey;
 `
 const CarouselBox = styled.div`
```

Comment:
Is this styled components?

Reply:
Yep. This shouldn't have been included in this branch; that was a mistake on my part.